sfm_pc/management/commands/import_google_doc.py

```diff
@@ -232,6 +237,8 @@ def handle(self, *args, **options):
             self.stdout.write('Loading data from Google Sheets...')
             all_sheets = self.get_sheets_from_doc(options['doc_id'], options['source_doc_id'])
 
+        self.country_code = options['country_code'].rstrip()
```

Comment:
Since we no longer technically require country code, how about  this?

```suggestion
        self.country_code = options.get('country_code', 'unnamed').rstrip()
```

Reply:
good idea